src/HttpUpgradeProtocolAccess.h

```diff
@@ -114,7 +114,14 @@ HttpUpgradeProtocolAccess::forApplicable(const ProtocolView &offer, const Visito
         seenApplicable = true; // may already be true
     }
     if (!seenApplicable && other) // OTHER is applicable if named rules were not
-        (void)visitor(ProtoOther, other);
+        (void)visitor(ProtoOther(), other);
+}
+
+const inline SBuf &
+HttpUpgradeProtocolAccess::ProtoOther()
+{
+    const static auto *b = new SBuf("OTHER");
+    return *b;
```

Comment:
Please follow [guideline style](https://wiki.squid-cache.org/DeveloperResources/SquidCodingGuidelines#rule-no-new-globals), at least for consistency sake:

```suggestion
    static const auto proto = new SBuf("OTHER");
    return *proto;
```

The above is also using an arguably more relevant/natural name for the variable than "b".